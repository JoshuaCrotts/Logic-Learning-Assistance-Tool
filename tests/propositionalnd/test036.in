// Note that this one does not work. what we may need to do is
// change the DeM finder to search on the conclusion as well, but
// only accept it if it's a goal AND a conclusion (if it is a C)?
~((A->B) & (B->A)) => ~(A <-> B)
