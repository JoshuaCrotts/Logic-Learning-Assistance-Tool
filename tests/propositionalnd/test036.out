1: ~((A -> B) & (B -> A))                            Ass.
2: ~(A <-> B)                                        DeM, ~((A -> B) & (B -> A))
∴ ~(A <-> B)  ■
